Userland/Libraries/LibGUI/RadioButton.cpp

```diff
@@ -21,8 +21,8 @@ RadioButton::RadioButton(String text)
 {
     set_exclusive(true);
     set_checkable(true);
-    set_min_width(32);
-    set_fixed_height(22);
+    set_min_size({ 22, 22 });
```

Comment:
Please show a comparason image of this change of min width (if this was intended), it seems it should be `{32, 22}` or something like that

Reply:
Make a list of these layout adjustments in the pull request with math so i can tell the widget layout didn't change without reason...

Reply:
Well, those were kind of arbitrary judgement calls.
The radio button was an exception because I thought people would expect them to be able to be square when they don't have a label.

Reply:
Let's be clear, that should be decided at some point. But it's kind of out of scope for this PR. They are only placeholders anyways, once calculated_min_size is implemented for a widget, there really is no point in having a fixed min_size as default.